com.pinegrow.Pinegrow.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: com.pinegrow.Pinegrow
+runtime: org.freedesktop.Sdk
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: pinegrow
+separate-locales: false
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+modules:
+
+  - name: pinegrow
+    buildsystem: simple
+    build-commands:
+      - install -D apply_extra /app/bin/apply_extra
+      - install -D pinegrow /app/bin/pinegrow
+      - install -Dm644 -t /app/share/metainfo com.pinegrow.Pinegrow.appdata.xml
+      - install -Dm644 -t /app/share/applications com.pinegrow.Pinegrow.desktop
+      - install -Dm644 -t /app/share/icons/hicolor/scalable/apps com.pinegrow.Pinegrow.svg
+    sources:
+      - type: extra-data
+        only-arches:
+          - x86_64
+        url: https://github.com/Pinegrow/PinegrowReleases/releases/download/pg7.4/PinegrowLinux64.7.4.zip
+        sha256: 3d3b6f4e2de620c69f54a99c68897eab1bf1f349a48615b934e0dafd919aeefc
+        size: 256110360
+        filename: Pinegrow.zip
+      - type: file
+        path: com.pinegrow.Pinegrow.appdata.xml
+      - type: file
+        path: com.pinegrow.Pinegrow.desktop
+      - type: file
+        path: com.pinegrow.Pinegrow.svg
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - set -e
+          - unzip Pinegrow.zip
```

Comment:
Can you redirect stdout to devnull? It's pretty noisy when installing via CLI right now.